### PR TITLE
fix Filter effect sounding different depending on sample rate

### PR DESCRIPTION
**Pull request by Be-ing**
The effect previously assumed a 44.1 kHz sample rate.

### CODE REVIEW

File: src/effects/native/filtereffect.cpp [hunk 1]
```diff
@@ -1,8 +1,10 @@
 #include "effects/native/filtereffect.h"
 #include "util/math.h"
 
-static const double kMinCorner = 0.0003; // 13 Hz @ 44100
-static const double kMaxCorner = 0.5; // 22050 Hz @ 44100
+namespace {
+static const double kMinCorner = 13; // Hz
+static const double kMaxCorner = 22050; // Hz
```
Comment (daschuer) [inline src/effects/native/filtereffect.cpp]: static keyword = old style anonymous namespace = new style. So static is redundant now.

**Review by daschuer — CHANGES_REQUESTED** [review]
What happen if we go from master to this version with the stored values? Are they clamped or back to default. Schould we take a new effect ID for it?

**Comment by Be-ing** [comment]
> What happen if we go from master to this version with the stored values? Are they clamped or back to default. Schould we take a new effect ID for it?

Parameter values are stored normalized to 0 through 1, so there is no issue.

**Comment by daschuer** [comment]
Thank you! LGTM

**Review by daschuer — APPROVED** [review]

**Pull request merged**